### DISCUSSION TITLE
Simple questions about Materials and visualization

### DISCUSSION THREAD

**Discussion by bosxered** [idaholab/moose#25222] [Q&A General] [answered]
Hello Moose experts,

I want to visualize **Material properties with higher resolution.**
I mean not like constant-order variables, but like first or second-order variables.
Is it possible?
![image](https://github.com/idaholab/moose/assets/79071957/55206cf0-5204-4703-bd95-e29867027567)

Actually, for higher resolution, I tried to follow https://mooseframework.inl.gov/syntax/Outputs/index.html but for me, material properties are gone in the ParaView. I posted some parts of my input file. Is it the wrong way to use refinements and displaced mesh?

```
[Materials]
  [./current_density_y]
    type = ParsedMaterial
    property_name = 'current_density'
    coupled_variables = 'Ey'
    material_property_names = 'electrical_conductivity'
    expression = 'electrical_conductivity*Ey'
    outputs = exodus
    use_displaced_mesh = true
  [../]
[]

[Outputs]
  exodus = true
  csv = true
  file_base = output

  [myexo]
    type = Exodus
    file_base = half
    interval = 2
#    start_step = 4340
#    refinements = 3
#    use_displaced = true
  []

  [myexo2]
    type = Exodus
    file_base = half-dis
    interval = 2
#    start_step = 4340
    refinements = 2
#    use_displaced = true
    show_material_properties = 'current_density'
  []

  [mymycp]
    type = Checkpoint
    num_files = 10
    interval = 50
    file_base = mycp
  []
[]
```

![image](https://github.com/idaholab/moose/assets/79071957/bb0ac3a6-1637-429e-8b57-2d97e7694fe6)
There is Current_density (Material properties), but a cell label (not point label) in the Paraview.

![image](https://github.com/idaholab/moose/assets/79071957/374174d6-2aa6-4c3d-bda4-f3cce0d48ba3)
No Material properties in the half-dis.e. I can't find them.

I think the refinement is working because... 
half-dis.e: 6.4MB
half.e: 3.4MB
output.e: 6.2MB.

Best Wishes,
Jeonghwan

**Comment by GiudGiud** [comment]
Hello

It's not impossible, it might not work without code changes. Let's try
You can try to use MaterialRealAux with a higher order monomial auxiliary variable.
You cannot use the short syntax in the Materials and Outputs blocks for higher order, you ll have to use the auxkernels system

Guillaume

**Comment by sourabhkadambi** [comment, marked as answer]
Hello. There's a filter called `Cell Data to Point Data` in ParaView which might be what you're looking to do. See https://vtk.org/doc/nightly/html/classvtkCellDataToPointData.html

**Reply by bosxered** [reply]
It's exactly what I wanted. Thanks a lot!
